2996_reflection/reflection.md

```diff
@@ -3365,21 +3443,18 @@ Add a new subsection of [expr.unary]{.sref} following [expr.delete]{.sref}
 ::: addu
 **The Reflection Operator   [expr.reflect]**
 
-FIXME: `$template-name$` and `$id-expression$` can both refer to template names, have to handle this better. See wording in the template argument parsing section.
-
 ```
 $reflect-expression$:
    ^ ::
-   ^ $nested-name-specifier$@~_opt_~@ $namespace-name$
-   ^ $nested-name-specifier$@~_opt_~@ $template-name$
-   ^ $nested-name-specifier$@~_opt_~@ $concept-name$
+   ^ $unqualified-id$
+   ^ $qualified-id$
    ^ $type-id$
-   ^ $id-expression$
+   ^ $pack-index-expression$
```

Comment:
Okay so before we had id-expression and then all the names separately. 

Now you removed the names and id-expression, and replaced it with... all the constituents of id-expression (unqualified-id, qualified-id, and pack-index-expression). Can we go back to id-expression then?

Why did we drop the namespace/template/concept name stuff? I don't think those are (un)qualified-ids.

Reply:
Add a `[...]{.ednote}` explaining this change.

Reply:
Done.

> I don't think those are (un)qualified-ids

Hmm. We'll have to see what Core says here.